content/pull-requests/collaborating-with-pull-requests/working-with-forks/detaching-a-fork.md

```diff
@@ -68,3 +68,6 @@ You can delete a fork and recreate the same repository, without the connection t
    cd ..
    rm -rf FORK-NAME.git
    ```
+
+## See also
```

Comment:
Unsure as to whether we want to add the link to Support here, but fixing the linter bug so that there's no test failure.

```suggestion
## See also

```